recipes/mattiasgustavsson-libs/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "20200514":
```

Sorry for not more specific in my comment on the issue

I could have provided a link to the wiki: https://github.com/conan-io/conan-center-index/wiki/FAQ#what-version-should-packages-use-for-libraries-without-official-releases

The date should be the date of the commit https://github.com/mattiasgustavsson/libs/commit/3da65d818fd4214d608d62666df32a0fc0428c80
```suggestion
  "20200423":
```